client/app/components/_createPickup/createPickup.html

```diff
@@ -26,12 +26,8 @@
       </md-input-container>
 
       <md-slider-container>
-        <label translate="CREATEPICKUP.MAX_COLLECTORS"></label>
-        <md-slider ng-model="$ctrl.pickupData.maxCollectors" min=1 max=9></md-slider>
-        <md-input-container>
-          <input name="MaxNoCollectors" ng-model="$ctrl.pickupData.maxCollectors"
-                 type="number" min=1 />
-        </md-input-container>
+        <label><span translate="CREATEPICKUP.MAX_COLLECTORS"></span>: {{$ctrl.pickupData.maxCollectors}}</label>
+        <md-slider md-discrete ng-model="$ctrl.pickupData.maxCollectors" step=1 min=1 max=9></md-slider>
```

Comment:
I think it's sometimes helpful to have more than 9 people picking up (big events), so I left the box there to enter it manually.